CHANGELOG.md

```diff
@@ -8,10 +8,13 @@ The format is based on [Keep a Changelog][keepachangelog] and this project adher
 
 ### Added
 
-- Added support for Keep-a-Changelog's links in the heading [#2]
-- Added support for using en dashes instead of minus signs [#2]
+- Added support for Keep-a-Changelog's links in the heading [#4]
+- Added support for using en dashes instead of minus signs [#4]
 - Added support for passing multiple parameters to `markdown-lint` [#3]
 
+[#3]:https://github.com/avto-dev/markdown-lint/pull/3
```

Comment:
I would place the links at the lower part of the file. Or directly after the link:

```markdown
Added support for passing multiple parameters to `markdown-lint` [#3](https://github.com/avto-dev/markdown-lint/pull/3)
```

The newline and then the link feels strange. (Because there are already links at the bottom and why the line break with 4 duplicated characters ( `[#3]`)

Reply:
:+1: